cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -8,7 +8,8 @@
 * license agreement from NVIDIA CORPORATION is strictly prohibited.
 */
 
-#include "../benchmarks/common/utils.hpp"
+#include "../include/claragenomics/cudapoa/utils.hpp" // for get_multi_batch_sizes()
```

Comment:
And this can just become `#include <claraparabricks/genomeworks/cudapoa/utils.hpp>`, much like thebutils included below